src/arch/runtime/event_queue/iocp.hpp

```diff
@@ -1,12 +1,15 @@
 #ifndef ARCH_RUNTIME_EVENT_QUEUE_IOCP_HPP_
 #define ARCH_RUNTIME_EVENT_QUEUE_IOCP_HPP_
 
+#include <stdint.h>
```

Comment:
So it should be changed in the rest of the codebase, then. It's _deprecated_ - there is a functional impact by using improper includes. It goes beyond style or convention...